modules/mvt/src/lib/binary-vector-tile/vector-tile-feature.js

```diff
@@ -0,0 +1,211 @@
+// This code is forked from https://github.com/mapbox/vector-tile-js under BSD 3-clause license.
```

Comment:
Nit: I would put the code that is significantly unique to us in a different folder from the more or less completely forked code.

Reply:
I would argue it makes sense to keep the classes together though, I feel that with the jsdocs it is now clear which files have the changes